lib/domain/user/i_profile_repository.dart

```diff
@@ -13,4 +14,7 @@ abstract class IProfileRepository {
     String? firstName,
     String? lastName,
   });
+  Future<Either<ProfileFailure, Unit>> updateCountry({
+    Location? location,
```

Comment:
Updating Country only takes a Country Code, so should be eg. `String countryCode`

Reply:
I got confused by swagger document result was coming back with both country name and code. I wasn't sure if request is done by only code or not. I will fix all the 3 country related comments.